public/helpers/update-users.js

```diff
@@ -0,0 +1,34 @@
+const updateUsers = (sender, users) => {
+  users.forEach((user) => {
+    if(user.permissions.length == 2){
+      if(mentorExist()){
+        addUser('moderator', user);
+      }
+      else{
+        addUser('mentor', user);
+      }
+    }
+    else{
+      addUser('students', user);
+    }
+  });
+};
+
+const mentorExist = () => {
+  return document.querySelector(".online__mentor-list").hasChildNodes();
+};
+
+const addUser = (type, user) => {
+  let className = createClassName(type);
+  const newUser = document.createElement("LI");
+  newUser.textContent = user.username;
+  newUser.classList.add("online__user");
+  let list = document.querySelector(className);
+  list.appendChild(newUser);
```

Comment:
Before this, we need to do this;
```javascript
list.innerHTML = ''
```
To clear all the users in the list items

Essentially, when this function `updateUsers` gets called, it'll rebuild the users every time